src/ddl/migrations/v4_renaming.sql

```diff
@@ -0,0 +1,99 @@
+
+-- drop VIEWs in `epidata` that act as aliases to (ie, they reference) VIEWs in `covid`
+USE epidata;
+DROP VIEW
+    epimetric_full_v,
```

Comment:
Should this be `signal_history_v`?

Reply:
hahahaha, good catch!  i ran the renaming commands on that migration script.  fixed now.  *facepalm*